include/faabric/scheduler/FunctionCallClient.h

```diff
@@ -14,6 +14,33 @@ using grpc::ClientContext;
 using grpc::Status;
 
 namespace faabric::scheduler {
+
+// -----------------------------------
+// Mocking
+// -----------------------------------
```

Comment:
The following functions are all used/ could be used for testing the client. I found it really difficult to test distributed RPCs without this